plasmapy/analysis/tests/test_fit_functions.py

```diff
@@ -44,7 +44,7 @@ def test_is_abc(self) -> None:
             ("root_solve", False),
         ],
     )
-    def test_methods(self, name, isproperty) -> None:
+    def test_methods(self, name: str, isproperty: bool) -> None:
```

Comment:
Is isproperty a boolean?

Reply:
Yep!  It is a `bool`.  This is a good one to double check.